tox.ini

```diff
@@ -58,7 +58,6 @@ deps =
 	-r{toxinidir}/requirements/quality.txt
 commands = 
 	pylint edx_rbac tests test_utils manage.py setup.py
-	pylint --py3k edx_rbac tests test_utils manage.py setup.py
```

Comment:
`--py3k` flag removed in `pylint==2.11.0` (https://github.com/PyCQA/pylint/blob/main/ChangeLog#L56)